java-surefire/src/test/resources/org/sonar/plugins/surefire/SurefireSensorTest/ignoreSuiteAsInnerClass/TEST-org.apache.shindig.protocol.TestHandler$Input.xml

```diff
@@ -1,68 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <testsuite errors="0" skipped="1" tests="1" time="0" failures="0" name="org.apache.shindig.protocol.TestHandler$Input">
   <properties>
-    <property value="Java(TM) 2 Runtime Environment, Standard Edition" name="java.runtime.name"/>
```

Comment:
While I understand the reasoning behind this removal : 
- I don't see why we should remove it : it was not a problem as far as I understand
- It is a valid report format/part and as such was a way to test the proper parsing of those report.

I don't think we should revert this but I tend to think it is a bit of a useless change. The only thing I would like to be sure of is that somewhere we have a small list of properties to ensure we parse it without issues.

Reply:
Agreed, let's discard it, it's working well with it and have been working for years. I removed the properties because it's completely ignored by the parser (and include some funny stuff such as the location of jdk on Simon's dropbox), but this part and all the possible properties are indeed some valid entries that we could encounter.